executor/common_linux.h

```diff
@@ -539,7 +539,10 @@ static void netlink_devlink_netns_move(const char* bus_name, const char* dev_nam
 	netlink_attr(&nlmsg, DEVLINK_ATTR_BUS_NAME, bus_name, strlen(bus_name) + 1);
 	netlink_attr(&nlmsg, DEVLINK_ATTR_DEV_NAME, dev_name, strlen(dev_name) + 1);
 	netlink_attr(&nlmsg, DEVLINK_ATTR_NETNS_FD, &netns_fd, sizeof(netns_fd));
-	netlink_send(&nlmsg, sock);
+	err = netlink_send(&nlmsg, sock);
+	if (err)
```

Comment:
This needs {} because we strip all debug calls during C repro generation. Usually it leads to broken source, but not in this case. Try:
```
$ cat /tmp/prog 
getpid()
$ syz-prog2c -prog /tmp/prog -enable=net_dev,devlink_pci -sandbox=none > /tmp/test.c
```

This results in:
```
  err = netlink_send(&nlmsg, sock);
  if (err)
  error:
    close(sock);
```

Reply:
Ah, will fix. Btw shouldn't "make presubmit" warn about this?

Reply:
I am all for it.
Try to add a test similar to this one:
https://github.com/google/syzkaller/blob/master/pkg/csource/csource_test.go#L165-L183
We already have the whole source in the commonHeader string variable. So potentially we just need to run some regexp on it? Something like ")\n\w*debug\("? Just a first wild guess. But you know what it should catch :)

Reply:
Done in https://github.com/google/syzkaller/commit/46869e3ee2274fbf353cb2c7d6cf36ed19e52b45
Found 2 more bugs